projects/storefrontstyles/scss/layout/page-templates/_product-detail.scss

```diff
@@ -15,7 +15,10 @@
 
```

Comment:
Please do the same for Line 14, in `3.0` is set as `grid-template-rows: repeat(2, auto) 1fr;`, therefore new property should be wrapped in a `forMixin` as well.

Reply:
Nevermind, found it.